### DISCUSSION TITLE
Is there a difference between Command and CommandArgs for client handlers?

### DISCUSSION THREAD

**Discussion by kchen1023** [cmu-sei/GHOSTS#372] [Q&A] [answered]
![image](https://github.com/cmu-sei/GHOSTS/assets/131231802/c6d20124-6be8-4d1a-96b0-244d7c0febac)

I know for other handlers, the command is specific for what we want to do (eg. command = create for word).
![image](https://github.com/cmu-sei/GHOSTS/assets/131231802/dad0d83a-e49f-4535-80b1-c033b9a47121)

But for CMD handler, what's the difference between Command and CommandArgs? In the example below, would there be a difference if we just wrote all the commands (or just one) in CommandArgs (i.e. having the cd %home... in CommandArgs instead of it being in Command)?
![image](https://github.com/cmu-sei/GHOSTS/assets/131231802/0b4be9e1-8bab-4b2c-8fed-e20a0644e993)

**Comment by sei-dupdyke** [comment, marked as answer]
You are correct: There is no difference for CMD, but there is for Word and some others.

Maybe not the best design decision, but it would be probably hard to change now.
